packages/publisher/src/generate-packages.ts

```diff
@@ -210,20 +210,19 @@ function dependencySemver(dependency: DependencyVersion): string {
   return dependency === "*" ? dependency : "^" + formatTypingVersion(dependency);
 }
 
-export function createNotNeededPackageJSON({ libraryName, license, fullNpmName, version }: NotNeededPackage): string {
+export function createNotNeededPackageJSON(pkg: NotNeededPackage): string {
   const out = {
-    name: fullNpmName,
-    version: String(version),
-    typings: null, // tslint:disable-line no-null-keyword
```

Comment:
Not only are the `typings` and `author` fields empty, they don't exist in the regular types package package.json, so I took the liberty of tidying up.